src/dispatch/case/flows.py

```diff
@@ -622,6 +622,9 @@ def case_create_resources_flow(
     if case.assignee:
         individual_participants.append((case.assignee.individual, None))
 
+    if case.reporter:
+        individual_participants.append((case.reporter.individual, None))
```

Comment:
It's used below in the function, like so:

```python
        # wait until all resources are created before adding suggested participants
        individual_participants = [x.email for x, _ in individual_participants]
        for email in individual_participants:
            # we don't rely on on this flow to add folks to the conversation because in this case
            # we want to do it in bulk
            case_add_or_reactivate_participant_flow(
                db_session=db_session,
                user_email=email,
                case_id=case.id,
                add_to_conversation=False,
            )
        # explicitly add the assignee to the conversation
        all_participants = individual_participants + [case.assignee.individual.email]
```